packages/select/src/index.ts

```diff
@@ -0,0 +1,4 @@
+export { Size } from './styleSets';
+export { default } from './Select';
```

Comment:
we can probably do this as a named export as well, since you can't use select without options. i think the benchmark was "can you use it alone?" in order for a component to be the default export alongside named exports in a package